cell_based/src/simulation/modifiers/ImmersedBoundarySvgWriter.hpp

```diff
@@ -0,0 +1,155 @@
+/*
```

Comment:
It makes sense for this writer to be a `modifier` rather than a `writer` does it?

Reply:
Keeping this as a modifier at least for now as it outputs a separate file each step, rather than writing a line into a particular file which is how the current writers are designed.